ArduPlane/mode_autoland.cpp

```diff
@@ -38,10 +38,20 @@ const AP_Param::GroupInfo ModeAutoLand::var_info[] = {
     // @Param: OPTIONS
     // @DisplayName: Autoland mode options
     // @Description: Enables optional autoland mode behaviors
-    // @Bitmask: 1: When set if there is a healthy compass in use the compass heading will be captured at arming and used for the AUTOLAND mode's initial takeoff direction instead of capturing ground course in NAV_TAKEOFF or Mode TAKEOFF or other modes.
+    // @Bitmask: 0: When set if there is a healthy compass in use the compass heading will be captured at arming and used for the AUTOLAND mode's initial takeoff direction instead of capturing ground course in NAV_TAKEOFF or Mode TAKEOFF or other modes.
     // @User: Standard
     AP_GROUPINFO("OPTIONS", 4, ModeAutoLand, options, 0),
 
+    // @Param: CLIMB
+    // @DisplayName: Climb before turning altitude
+    // @Description: Vehicle will climb with limited turn ability (LEVEL_ROLL_LIMIT) upon mode entry to this altitude, before proceeding to loiter-to-alt and landing legs. If a value less than AUTOLAND_WP_ALT is entered, AUTOLAND_WP_ALT will be used. If TERRAIN is enabled and healthy, the altitude is above terrain, otherwise above HOME.
```

Comment:
Sorry to seem to be nitpicking @Hwurzburg but should this only apply if TERRAIN_FOLLOW is enabled - either all or for AUTOLAND? (which would mean adding a bit for AUTOLAND to the bit mask).

Reply:
not necessarily, but the mode needs a bit in the follow table anyway, so if terrain following, the climb will be above terrain....the final WP alt MUST always be above home, lest you crash into a gully before the runway...